sentry-sample/src/main/java/io/sentry/sample/MainActivity.java

```diff
@@ -58,6 +60,56 @@ protected void onCreate(Bundle savedInstanceState) {
           }
         });
 
+    binding.sessions.setOnClickListener(
+        view -> {
+          for (int i = 0; i < 100; i++) {
+            System.out.println("session start: i = " + i);
```

Comment:
H: Is this change related to the session data generator? Is this going to land in this PR?

Reply:
related to session data generator, yes, land in this PR, no

Reply:
why is this in the diff?

Reply:
aka Draft PR, we agreed to lift a draft PR to share the knowledge of the session data generator (how we did it), this was asked by you btw on a daily standup.

Reply:
It's fine that it's draft, but it was unclear if this is planned to be merged or not.
The "refactors" like `@Null` and `final` I don't expect would be deleted once the PR drops the Draft, or would it?

Sorry if it was too detailed for a Draft PR. I won't review this a third time as Draft then. I'll wait until we can get closer to review it.

Reply:
clarified offline